databroker/cache.go

```diff
@@ -36,11 +37,10 @@ type DataBroker struct {
 	manager          *manager.Manager
 	eventsMgr        *events.Manager
 
-	localListener                net.Listener
-	localGRPCServer              *grpc.Server
-	localGRPCConnection          *grpc.ClientConn
-	dataBrokerStorageType        string // TODO remove in v0.11
-	deprecatedCacheClusterDomain string // TODO: remove in v0.11
```

Comment:
did we remove all references to `dataBrokerStorageType` and `deprecatedCacheClusterDomain`; should we mention that in the changelog / upgrade guide or no?

Reply:
they were private. I'm not sure what could've referenced them.